samples/advanced/content-security-policy.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=Edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta http-equiv="Content-Security-Policy" content="default-src 'self'">
+	<title>Scriptable > Bubble | Chart.js sample</title>
+	<link rel="stylesheet" type="text/css" href="../../dist/Chart.min.css">
+	<link rel="stylesheet" type="text/css" href="./content-security-policy.css">
```

Comment:
This confused me at first. Because the CSS file has "content-security-policy" in the name I thought that it was specifically demonstrating an interaction with the content security policy. Maybe "example.css" or "advanced.css" would remove the potential for confusion?

Reply:
I don't think it's a source of confusion when you known how CSP is working and I prefer to keep the `.js` and `.css` matching the `.html` file name. Actually, having to move the CSS and JS code in separate files is required by the `default-src 'self'` CSP.

Reply:
I think the note that was added helps a lot too, so renaming probably not needed anymore